lib/fse.h

```diff
@@ -45,6 +45,32 @@ extern "C" {
 #include <stddef.h>    /* size_t, ptrdiff_t */
 
 
+/*-*****************************************
+*  FSE_PUBLIC_API : control library symbols visibility
+******************************************/
+#if defined(__GNUC__) && (__GNUC__ >= 4)
+#  define FSE_PUBLIC_API __attribute__ ((visibility ("default")))
```

Comment:
This works fine for libfse,
but in the context of libzstd, it means fse symbols will also get automatically exposed, which is precisely what I would like to avoid.

Any solution that would make fse symbols remain internal on libzstd, and exposed on libfse, will be welcomed.
It would ease maintenance, by avoiding the need to separate 2 different set of files.